libraries/testkit/cluster.py

```diff
@@ -133,6 +152,8 @@ def reset(self, sg_config_path):
         server_port = 8091
         server_scheme = "http"
         couchbase_server_primary_node = add_cbs_to_sg_config_server_field(self._cluster_config)
+        if is_ipv6:
```

Comment:
The usage of is_ipv6 is is_ipv6(cluster_config). Did this check work?

Reply:
wrong implementation, but it would be true for IPv6 and IPv4 too. Fixing it.